python/paddle/fluid/incubate/fleet/parameter_server/distributed_transpiler/__init__.py

```diff
@@ -48,10 +48,11 @@ def init_worker(self):
         Returns:
             None
         """
-        pass
+        if not self._transpile_config.sync_mode:
+            self._communicator = Communicator(self._main_program)
+            self._communicator.start()
 
-    def run_worker(self, main_programs=None, scopes=None):
-        pass
+        self._executor.run(self._startup_program)
```

Comment:
Should not run the startup program in `init_worker `. 
For training, convert a local training to fleet training neither delete code nor change method behavior.